internal/detector/jar_assessor.go

```diff
@@ -50,23 +51,20 @@ func NewJarAssessor(jarChecker JarChecker) JarAssessor {
 	}
 }
 
-func (ja *JarAssessor) Assess(path string) (JarAssessement, error) {
+func (ja *JarAssessor) Assess(path string) (*JarAssessement, error) {
```

Comment:
opinion: instead of returning nil if log4j wasn't found I'd add a `ContainsLog4J` (or similar) property to `JarAssesement`. Nils require a check on the caller side and can lead to npd if forgotten

Reply:
done